drivers/gles3/shaders/scene.glsl

```diff
@@ -263,7 +263,7 @@ void light_compute(
 
 uniform highp mat4 refprobe1_local_matrix;
 out mediump vec4 refprobe1_reflection_normal_blend;
-uniform highp vec3 refprobe1_box_extents;
+uniform highp vec3 refprobe1_box_size;
```

Comment:
Instead of converting the shader to use use size, you should convert from size to extents before sending to the shader. 

That being said, this code is all unused right now and will be rewritten soon. So for now you can just revert all the changes to this file